docs/install.md

```diff
@@ -5,21 +5,23 @@ title: "Installation"
 
 Verdaccio is a multiplatform web application. To install it, you need a few basic prerequisites.
 
-#### Prerequisites
+## Prerequisites
 
-1. Node higher Node `10.x` (LTS "Carbon") is the minimum supported version.
+1. **Node.js** `v10.x (LTS "Dubnium")` or higher.
 
-> After v4.5.0 *Node.js v10* is now the minimum supported version. If you cannot upgrade *Node.js v8*, keep using `v4.4.4`.
+> Verdaccio >v4.5.0, requieres *Node.js >v10*. If you can not upgrade from *Node.js v8*, you can keep using Verdaccio `v4.4.4`.
```

Comment:
> After v4.5.0 *Node.js v10* is now the minimum supported version. If you cannot upgrade *Node.js v8*, keep using `v4.4.4`.

Actually this line is incorrect, does not apply anymore.

Verdaccio 4 supports Node v8.  Can we remove this?

Reply:
Thanks for the clarification. ;-)